### ISSUE TITLE
Every error is printed twice

### ISSUE THREAD

**Issue by navidshaikh** [knative/client#89]
```
kn service list
Error: Get https://192.168.39.212:8443/apis/serving.knative.dev/v1alpha1/services: dial tcp 192.168.39.212:8443: connect: no route to host
Get https://192.168.39.212:8443/apis/serving.knative.dev/v1alpha1/services: dial tcp 192.168.39.212:8443: connect: no route to host
```

**Comment by maximilien** [comment]
This sucks and I think has to do with the fact that we don't wrap errors. My hunch. Did you plan to fix this @navidshaikh? Otherwise I can take a crack at it. LMK.

**Comment by maximilien** [comment]
Sweet @navidshaikh, will do. I'll update here if there are items to discuss before PR. Cheers 🍻

**Comment by maximilien** [comment]
Actually it's a simpler change. Submitting PR now.

**Comment by navidshaikh** [comment]
Fixed by #96 , thanks @maximilien !
